src/libcore/macros.rs

```diff
@@ -519,54 +519,69 @@ macro_rules! unreachable {
     });
 }
 
-/// Indicates unfinished code.
+/// Indicates unfinished code by panicking with a message of "not yet implemented".
 ///
-/// This can be useful if you are prototyping and are just looking to have your
-/// code type-check, or if you're implementing a trait that requires multiple
-/// methods, and you're only planning on using one of them.
+/// This is useful if you are prototyping and want your program to type-check
+/// and stop execution whenever it reaches logic that has not yet been implemented.
 ///
 /// # Panics
 ///
-/// This will always [panic!](macro.panic.html)
+/// This will always [panic!](macro.panic.html) because `unimplemented!` is just a
+/// shorthand for `panic!` with a fixed, specific message.
```

Comment:
(subjective:) I don't really see the use of elaborating this sentence, and so I personally would leave it as it was before.

Reply:
Up to you if you'd like me to remove this. For me, knowing that `unimplemented!` is really just `panic!` with a default message makes me understand there isn't any extra magic going on here.